comps/third_parties/sglang/src/Dockerfile

```
@@ -27,20 +27,29 @@ RUN curl -fsSL -v -o miniforge.sh -O https://github.com/conda-forge/miniforge/re
     bash miniforge.sh -b -p ./miniforge3 && \
     rm miniforge.sh
 
-RUN git clone https://github.com/jianan-gu/sglang -b llama4_optimzed_cpu_r1
+RUN git clone https://github.com/jianan-gu/sglang --depth=1 -b llama4_optimzed_cpu_r1 && \
+    cd /root/sglang  && rm -rf .git
```

Simpler just not to use `cd`:
```suggestion
RUN git clone https://github.com/jianan-gu/sglang --depth=1 -b llama4_optimzed_cpu_r1 && \
    rm -rf sglang/.git
```